hack/pre-release.sh

```diff
@@ -9,6 +9,8 @@ source $WMCO_ROOT/hack/common.sh
 CVP=false
 BASEIMAGE=false
 OPERATOR_SDK_VERSION=v1.15.0
+PINK='\033[95m'
+RESET='\033[0m'
```

Comment:
Nit: You are assuming the `RESET` color instead of figuring out what the current color is. I could be using a different theme that does not use the `RESET` color by default.